sw/device/tests/alert_handler_lpg_sleep_mode_pings.c

```diff
@@ -510,6 +509,18 @@ bool test_main(void) {
 
   init_test_components();
 
+  dif_rstmgr_reset_info_bitfield_t rst_info = rstmgr_testutils_reason_get();
+  rstmgr_testutils_reason_clear();
+
+  // We need to initialize the info FLASH partitions storing the Creator and
+  // Owner secrets to avoid getting the flash controller into a fatal error
+  // state.
+  if (kDeviceType == kDeviceFpgaCw310 && rst_info & kDifRstmgrResetInfoPor) {
+    CHECK_STATUS_OK(keymgr_testutils_flash_init(&flash_ctrl, &kCreatorSecret,
+                                                &kOwnerSecret));
```

Comment:
Is this something that needs to be added to other testcases, too? E.g. https://github.com/lowRISC/opentitan/issues/16822?

Reply:
I will take a look.

Reply:
Added one more commit to resolve https://github.com/lowRISC/opentitan/issues/16576, and marked https://github.com/lowRISC/opentitan/issues/16822 as obsolete.